src/components/Send.jsx

```diff
@@ -640,7 +640,11 @@ export default function Send() {
                         {settings.useAdvancedWalletMode
                           ? t('send.account_selector_option_dev_mode', { number: account })
                           : t('send.account_selector_option', { number: account })}{' '}
-                        {settings.showBalance && `(\u20BF${balance})`}
+                        {settings.showBalance
+                          ? settings.unit === 'sats'
+                            ? `(${formatSats(balance * 100000000)} sats)`
+                            : `(\u20BF${formatBtc(balance)})`
+                          : '*****'}
```

Comment:
Let's just remove that case for when the blance is hidden. Will look best if we just don't show anything there like before.